lib/task1.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+var readline = require('readline');
+var rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  terminal: false
+});
+try {
+  rl.on('line', function (line) {
+    console.log(line.split('').reverse().join(''));
+  });
+} catch (error) {
+  console.log(error);
+}
```

Comment:
this catch block is not going to work. you need to use `.on('error', ...)` callback